### PR TITLE
feat: docker, eslint/prettier, projet and !par and !vanc command

### PR DESCRIPTION
**Pull request by sanghin**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,10 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+end_of_line = lf
+max_line_length = 100
```
Comment (ltribolet) [inline .editorconfig]: 120, we are blocked in 1995

File: server.js [hunk 2]
```diff
@@ -1,77 +1,106 @@
...
+      if (!reply) {
+        // link is fresh or at least 12 hours old
+        // save the hash and links meta in redis for 12 hours
+        const data = {
+          user: message.author.username,
+          channel: message.channel.name,
+          date: message.createdAt,
+        };
+        redisClient.set(hashedUrl, JSON.stringify(data), 'EX', TWELVE_HOURS);
+      } else if (message.deletable) {
```
Comment (ltribolet) [inline server.js]: Break if / else if as just two if because it's not related, plus it's ugly.

File: server.js [hunk 3]
```diff
@@ -1,77 +1,106 @@
...
-    }
+  if (message.content.replace(/\s/g, '').match(IS_NESTCEPAS_REGEX)) {
+    message.channel.send('', { file: `${PATH_TO_STATIC_FOLDER}/nestcepas.gif` });
+  }
+
+  if (message.content.replace(/\s/g, '').match(IS_PROJET_REGEX)) {
+    message.channel.send('', { file: `${PATH_TO_STATIC_FOLDER}/projet.gif` });
+  }
+
+  if (message.content.match(IS_GET_VANCOUVER_TIME_REGEX)) {
```
Comment (ltribolet) [inline server.js]: I think we can make a function as it is repeated many times ?

File: server.js (hunk 2 above)
Comment (sanghin) [reply server.js]: if you break this into 2 separate if, you will delete the message everytime someone is posting a URL.

File: server.js (hunk 3 above)
Comment (sanghin) [reply server.js]: Yup totally, and we should add some test too. I want to add some CI server with unit testing etc. I know that's totally overkill but i think it's a good way to learn 😄

File: .editorconfig (hunk 1 above)
Comment (sanghin) [reply .editorconfig]: Not everyone has a 4K 120 inches screen 😛

File: server.js (hunk 2 above)
Comment (sanghin) [reply server.js]: But i'll agree with you, this parts need some "love" 😄

**Pull request merged**